store/pizza-slice.ts

```diff
@@ -14,7 +15,11 @@ const slice = createSlice({
   initialState,
   reducers: {
     onSave: (state, action: PayloadAction<IPizza[]>) => {
-      state.list = action.payload 
+      state.list = action.payload
+      state.filteredList = action.payload
+    },
+    pizzasCategorySelector: (state, action: PayloadAction<Categories>) => {
```

Comment:
better to do filtering in selector and not in reduce, cause you are manipulating already existing data